qutebrowser/mainwindow/tabbedbrowser.py

```diff
@@ -412,6 +412,8 @@ def tabopen(self, url=None, background=None, explicit=False):
             background = config.get('tabs', 'background-tabs')
         if not background:
             self.setCurrentWidget(tab)
+        elif background:
```

Comment:
Since there is `if not background` above, this could be cleaned up like this:

```
if background:
    self.tab_index_changed.emit(self.currentIndex(), self.count())
else:
    self.setCurrentWidget(tab)
```